### PR TITLE
chore: release tasks for extension version 0.4.1

### PR DESCRIPTION
**Pull request by jgpruitt**

### CODE REVIEW

File: projects/extension/RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+  default. An `exact_count` parameter was added. If true, the original behavior is used. 
+  c11db9c2d7fb8346f28f4de17bf3706e9d1620d4
+- If a vectorizer has no queue table, or the user does not have select privileges on the queue table
+  we will now return null for the pending_items column in the vectorizer_status view. 
+  f17d1b908df9fd7072b5554de7dc162102a5611b
+
+### Deprecations
+
+- Versions `0.1.0`, `0.2.0`, and `0.3.0` are deprecated and will be removed in a future release.
+  You must `DROP EXTENSION ai` and then `CREATE EXTENSION ai VERSION '0.4.1' CASCADE`.
```
Comment (JamesGuthrie) [inline projects/extension/RELEASE_NOTES.md:70]: This sounds like you need to do this _also_ to upgrade from 0.4.0

Comment (jgpruitt) [reply projects/extension/RELEASE_NOTES.md:70]: `projects/extension/sql/ai--0.4.0--0.4.1.sql` allows for upgrades from `0.4.0` to `0.4.1`

Comment (JamesGuthrie) [inline projects/extension/RELEASE_NOTES.md:70]: I am aware. My point was that it's not necessarily clear that the advice in the second sentence only applies to the first sentence. My suggestion would be:

```suggestion
  To upgrade from a deprecated version, you must `DROP EXTENSION ai` and then `CREATE EXTENSION ai VERSION '0.4.1' CASCADE`.
```

**Review by JamesGuthrie — APPROVED** [review]

File: projects/extension/RELEASE_NOTES.md (hunk 1 above)
Comment (jgpruitt) [reply projects/extension/RELEASE_NOTES.md:70]: Ahhh. okay. I gotcha.

**Pull request merged**